### PR TITLE
Make default symbol table conform to the spec

### PR DESCRIPTION
**Pull request by divarvel**
See https://github.com/biscuit-auth/biscuit/issues/94

### CODE REVIEW

**Comment by Korbik** [comment]
We should add a test in the biscuit's specs samples.

**Comment by divarvel** [comment]
Yes, I'm waiting for consensus on where to go. Once we have it we can cover this case in the samples

**Comment by KannarFr** [comment]
It still needs some updates in the binary files. Waiting for symbols spec and I think @Geal will push binary tests files to spec accordingly. Then I'll bump them here.

**Comment by divarvel** [comment]
I'll close this PR in favour of your work

**Comment by KannarFr** [comment]
LGTM, test17 fails due to missing `Add` on `String` and test23 due to the authorization scope not being implemented.

@Korbik @Geal @divarvel can we merge this?

File: src/main/java/com/clevercloud/biscuit/datalog/SymbolTable.java [hunk 1]
```diff
@@ -14,17 +13,32 @@
...
       return Instant.ofEpochSecond(epochSec).atOffset(ZoneOffset.ofTotalSeconds(0)).format(dateTimeFormatter);
    }
+
+   /**
+    * Due to https://github.com/biscuit-auth/biscuit/blob/master/SPECIFICATIONS.md#symbol-table,
+    * We need two symbols tables:
+    * * one for the defaults symbols indexed from 0 et 1023 in <code>defaultSymbols</code> list
+    * * one for the usages symbols indexed from 1024 in <code>symbols</code> list
+    */
+   public final List<String> defaultSymbols;
```
Comment (Geal) [inline src/main/java/com/clevercloud/biscuit/datalog/SymbolTable.java:29]: maybe this could be a static list?

**Review by Geal — APPROVED** [review]

File: src/main/java/com/clevercloud/biscuit/datalog/SymbolTable.java (hunk 1 above)
Comment (KannarFr) [reply src/main/java/com/clevercloud/biscuit/datalog/SymbolTable.java]: done in 6138079.

Comment (Geal) [reply src/main/java/com/clevercloud/biscuit/datalog/SymbolTable.java]: nice!

**Pull request merged**